tutorial/admin/README.md

```diff
@@ -27,29 +26,31 @@ Table of Contents
 10. [Restrict maximum trace file size (TBD)](#10-restrict-the-maximum-size-of-trace-files)
 
 ## 0. Verify GEOPM pre-requisites
-Run `./test_prereqs.sh` on a compute node in the context of a regular user
-to ensure the GEOPM components are installed and functioning properly.
-In the event that something is misconfigured, an ERROR or WARNING message
-will be displayed recommending a course of action to address the issue
-or refer the user back to the main GEOPM README for guidance.
+Run [`./test_prereqs.sh`](./test_prereqs.sh) on a compute node as an
+unprivileged user to ensure the GEOPM components are installed and functioning
+properly. In the event that something is misconfigured, an ERROR or WARNING
+message will be displayed recommending corrective action or referring the user
+back to the main GEOPM README for guidance.
 
 ## 1. GEOPM+SLURM static policy plugin
-To setup a system with the GEOPM+SLURM static policy plugin, the libgeopmpolicy
-must be available.  GEOPM provides two libraries. The first, libgeopmpolicy,
+GEOPM provides two libraries: `libgeopmpolicy` and `libgeopm`. `libgeopmpolicy`
 contains tools for interacting with hardware signals and controls, such as
-`geopmread`, and the supporting library functions.  The second, libgeopm,
-contains all the same functions and additionally provides tools for launching
-and interacting with MPI applications. The SLURM plugin requires libgeopmpolicy
-only; it does not use libgeopm or MPI.  If the GEOPM runtime will be installed
-(libgeopm and MPI launch tools), it should be installed in a different
-location, preferably using the module system.  The packages are available
-through OpenHPC (https://openhpc.community/downloads/).  See step 2 in this
-guide.
-
-A.  To build libgeopmpolicy and a compatible libgeopm_slurm.so:
-* Obtain the source code for geopm from https://github.com/geopm/geopm
-* Make sure no modules are loaded in the environment that will affect
-  compilation:
+`geopmread`, and the supporting library functions. `libgeopm` contains all of
+these functions and also provides tools for launching and interacting with MPI
+applications.
+
+The `libgeopmpolicy` library must be available in order to set up a system with
+the GEOPM+SLURM static policy plugin. The SLURM plugin requires
+`libgeopmpolicy` alone; it does not use `libgeopm` or MPI.  If the GEOPM
+runtime will be installed (including `libgeopm` and MPI launch tools), it
+should be installed in a different location, preferably using the module
+system. The packages are available through OpenHPC
+(<https://openhpc.community/downloads/>).  See step 2 in this guide.
```

Comment:
It wasn't clear to me if "step 2" referred to step two in this document or some other document. A link would be helpful to resolve this.

Reply:
It should point at the "Runtime Capabilities" section.  A link with the name would be better in case we renumber the sections.

Reply:
Done.